packages/angular/src/generators/remote/schema.d.ts

```diff
@@ -28,4 +28,5 @@ export interface Schema {
   skipFormat?: boolean;
   standalone?: boolean;
   ssr?: boolean;
+  typescriptConfiguration?: boolean;
```

Comment:
Why is this not called `useTypescriptConfiguration`?

It feels like the field should point to the configuration file, but it's just a boolean flag.

I guess this applies to other fields above (`ssr`, `standalone`) as well. If we want to match the same style, we could name this one `typescript`.

Reply:
It's to match the flag on the react generator which is also `typescriptConfiguration`.

This was previously `typescript`, however, in react land it's more confusing because it could be an indication to use TS with your react app, rather than specifically about the configuration file.